apps/docs/pages/guides/realtime/quotas.mdx

```diff
@@ -0,0 +1,112 @@
+  Upgrade your plan to increase your quotas. Without a spend cap, or on an Enterprise plan, some
+  quotas are still in place to protect budgets. All quotas are configurable per project. [Contact
+  support](https://app.supabase.com/support/new) if you need your quotas increased. Our cluster
+  supports millions of concurrent connections and message throughput for production workloads.
+</Admonition>
+
+## Quotas by Plan
+
+You can increase your quotas by upgrading your plan and removing your spend cap. We still have a starting quota set even without a spend cap to protect your budget just in case.
+
+If you need your quotas increased further please [contact support](https://app.supabase.com/support/new).
+
+### Free
+
+- 200 concurrent clients
+- 100 messages per second
+- 100 Channel joins per second
+
+### Pro
+
+- 500 concurrent clients
+- 500 messages per second
+- 500 Channel joins per second
+
+### Pro with no spend cap
+
+Starts at:
+
+- 10,000 concurrent clients
+- 2,500 messages per second
+- 2,500 Channel joins per second
+
+### Enterprise
+
+Starts at:
+
+- 10,000 concurrent clients
+- 2,500 messages per second
+- 2,500 Channel joins per second
+
+## System Quotas
+
+The following quotas apply to all projects:
+
+- 100 Channels per connected client
+
+## Client-Side Limiting
+
+Some basic WebSocket message rate limiting is implemented client-side.
+
+For example, the [multiplayer.dev demo](/docs/guides/realtime/quickstart#cursor-positions) instantiates the Supabase client with an `eventsPerSecond` parameter.
+
+## Quota Errors
+
+When you reach a quota errors can appear in backend logs and messages in the WebSocket connection.
+
+<Admonition type="note">
+  Use the [Realtime Inspector](https://realtime.supabase.com/inspector/new) to reproduce an error
+  and share those connection details with Supabase support.
+</Admonition>
+
+### Backend Logs
+
+If your project is being limited by a quota, check your [Realtime logs](https://app.supabase.com/project/_/database/realtime-logs).
+
+### WebSocket Errors
+
+- `tenant_events`: Clients will be disconnected if your project is generating too many messages per second. `supabase-js` should reconnect automatically when the message throughput decreases below your plan quota.
+
+<Admonition type="note">
+  An `event` is a WebSocket message delivered to, or sent from a client.
+</Admonition>
+
+Some quotas can cause a Channel join to be refused. Realtime will reply with one of the following WebSocket messages:
+
+- `too_many_channels`: Too many channels currently joined for a single client.
+- `too_many_connections`: Too many total concurrent connections for a project.
+- `too_many_joins`: Too many Channel joins per second.
+
+<Admonition type="note">
+  Use your browser's developer tools to find the WebSocket initiation request and view individual
+  messages.
+</Admonition>
+
+## Postgres Changes Payload Quota
+
+Realtime Postgres Changes has a change payload size quota of 1 mebibyte (MiB).
```

Comment:
```suggestion
Realtime Postgres Changes has a change payload size quota of 1 megabyte (MiB).
```

Reply:
Any reason this isn't included in the "Quotas by Plan" section? Seems a bit hidden away